src/page-modules/contact/layout.tsx

```diff
@@ -0,0 +1,19 @@
+import { HomeLinkWithGlobalMessageLayout } from '@atb/layouts/shared/home-link-with-global-message-layout';
+import { PropsWithChildren } from 'react';
+import { ErrorContent } from '../error';
+
+export type ContactLayoutProps = PropsWithChildren<{}>;
+
+function ContactLayout({ children }: ContactLayoutProps) {
+  const contactPage = process.env.NEXT_PUBLIC_CONTACT_PAGE;
+  if (contactPage) {
+    return (
+      <HomeLinkWithGlobalMessageLayout>
+        {children}
+      </HomeLinkWithGlobalMessageLayout>
+    );
+  }
+  return <ErrorContent statusCode={404} />;
```

Comment:
I think this could be moved out of the layout and handled by the `getServerSideProps` function on the contact page (`src/pages/contact/index.tsx`). See suggested change below